ft232/libftdi.py

```diff
@@ -206,7 +206,7 @@ def _setDataCharacteristics(self):
     def setByteSize(self, bytesize):
         """Change byte size."""
         if bytesize not in self.BYTESIZES: raise ValueError("Not a valid byte size: %r" % bytesize)
-        self._bytesize = bytesize0
+        self._bytesize = bytesize
```

Comment:
Ditto about the bug fix, please send it as a separate PR (it could be included as a second patch with the fix to the writetimeout).